cypress/commons/actions/generic/TableParameters.js

```diff
@@ -11,6 +11,36 @@ function getGrid(tableParameterElement) {
   return tableParameterElement.find(GENERIC_SELECTORS.genericComponents.table.grid);
 }
 
+function getErrorPanel(tableParameterElement) {
```

Comment:
`getErrorPanel` should be `getErrorsPanel` to be consistent with the other functions names